R/assertions.R

```diff
@@ -1157,7 +1206,11 @@ assert_unit <- function(dataset,
       class = c(class, "assert-admiraldev")
     )
   }
-  if (tolower(units) != tolower(required_unit)) {
+
+  if (!is.null(required_unit) && !(tolower(units) %in% tolower(required_unit))) {
+    # change cli `.val` to end with OR instead of AND
+    divid <- cli_div(theme = list(.val = list("vec-last" = ", or ", "vec_sep2" = " or ")))
+
```

Comment:
```suggestion
```


I believe this is not needed if you use `.or` instead of `.val` in the message below.

Reply:
See above

Reply:
Not relevant anymore.